tests/baselines/reference/namedTupleMembersErrors.types

```diff
@@ -1,21 +1,21 @@
 === tests/cases/conformance/types/tuple/named/namedTupleMembersErrors.ts ===
-export type Segment1 = [length: number, number]; // partially named, disallowed
->Segment1 : [number, number]
+export type Segment1 = [length: number, number];
+>Segment1 : [length: number, number]
 
-export type Segment2 = [number, size: number]; // partially named, disallowed
->Segment2 : [number, number]
+export type Segment2 = [number, size: number];
+>Segment2 : [length: number, number]
 
-export type List = [item: any, ...any];  // partially named, disallowed
+export type List = [item: any, ...any]; 
 >List : [any, ...any[]]
 
-export type List2 = [any, ...remainder: any];  // partially named, disallowed
+export type List2 = [any, ...remainder: any]; 
 >List2 : [any, ...any[]]
 
-export type Pair = [item: any, any?];  // partially named, disallowed
->Pair : [any, any?]
+export type Pair = [item: any, any?]; 
+>Pair : [item: any, any?]
 
-export type Pair2 = [any, last?: any];  // partially named, disallowed
->Pair2 : [any, any?]
+export type Pair2 = [any, last?: any]; 
+>Pair2 : [item: any, any?]

```

Comment:
This looks incorrect, it should be:
```suggestion
export type Pair2 = [any, last?: any]; 
>Pair2 : [any, last: any?]
```

The same goes for some of the other ones in this file. Yea, I know it's still a draft - I'm sorry if I'm commenting on it too soon :P

Reply:
Aha, thank you for pointing this out - I might not have caught it before un-drafting!

Fixed in 2c316ce. The issue was in generating the cache key for tuple names. I'd changed the logic to bail out if any member is anonymous. That meant that all mixed tuples received the same key.

Reply:
Ah, but then `asyncFunctionContextuallyTypedReturns` switches back to the `Promise<[0] | [0]>`... fun!

Reply:
That was because the caching logic was checking `namedMemberDeclarations.length` for existence instead of checking for any defined elements of that array. The tuple `[0]` ends up with `[undefined]`. Fixed in d246ce999b